src/createServer.js

```diff
@@ -1,11 +1,25 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+
+const usersRouter = require('./routes/users');
+const expensesRouter = require('./routes/expenses');
+
+const { usersService } = require('./services/users');
+const { expensesService } = require('./services/expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  usersService.reset();
+  expensesService.reset();
+
+  const app = express();
+
+  app.use(cors());
+  app.use(usersRouter);
+  app.use(expensesRouter);
```

Comment:
I think better move `'/users'` here, then you can delete it from `./routes/users`

```suggestion
  app.use('/users', usersRouter);
```
And same for app.use(expensesRouter);

Reply:
I know, but I did so because App has a shorter code now, I dont know how correct it is I'll waiting for a mentor review 🙂

Reply:
it's better move default routes here in this case you can follow DRY in `users` & `expenses `routes